src/actions.js

```diff
@@ -188,6 +189,73 @@ export const removePromoCode = () => (dispatch, getState) => {
     }
 }
 
+export const validatePromoCode = (ticketData) => async (dispatch, getState, { apiBaseUrl, getAccessToken }) => {
+
+    const { registrationLiteState: { settings: { summitId }, promoCode: currentPromoCode } } = getState();    
+
+    if (summitId && currentPromoCode) {
+
+        dispatch(startWidgetLoading());
+
+        const { ticketQuantity, id, sub_type } = ticketData;
+
+        const access_token = await getAccessToken();
+
+        let params = {
+            access_token,
+            filter: `ticket_type_id==${id},ticket_type_qty==${ticketQuantity},ticket_type_subtype==${sub_type}`
```

Comment:
this is an OR
u need an AND
use uri JS
should be ?filter[]= ticket_type_id==123&filter[]= ticket_type_qty==123&filter[]=ticket_type_subtype==123